ccd-definition/AuthorisationCaseField/CareSupervision/solicitor.json

```diff
@@ -3,8 +3,14 @@
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "CARE_SUPERVISION_EPO",
     "CaseFieldID": "caseHistory",
-    "UserRole": "[SOLICITOR]",
-    "CRUD": "CRU"
+    "AccessControl": [
```

Comment:
just 
```
"UserRoles": ["[SOLICITOR]"],
"CRUD": "CRU"

should do the job

but anyway i though that it is about adding new roles as well
```
"UserRoles": ["[SOLICITOR1]", "[SOLICITOR2]",...."[SOLICITOR10]"],
    "CRUD": "CRU"

Reply:
"UserRoles": ["[SOLICITOR]"] implementation is done.

One thing we discussed and tested is that numbers are not allowed in role identifier. [SOLICITOR1], [SOLICITOR_1] etc won't work as '_' and '1' are not allowed. The validation accepts alpha characters only.

The proposed change is to use [SOLICITORONE], [SOLICITORTWO].... unless there are other ideas.